integration-tests/basic_contract_ref/lib.rs

```diff
@@ -0,0 +1,40 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+#[ink::contract]
+mod basic_contract_ref {
+    /// We import the generated `ContractRef` of our other contract.
+    ///
+    /// Note that the other contract must have re-exported it (`pub use OtherContractRef`) for us
+    /// to have access to it.
+    use other_contract::OtherContractRef;
+
+    #[ink(storage)]
+    pub struct BasicContractRef {
```

Comment:
Similarly here `BasicContractRef` is confusing. 
```suggestion
    pub struct BasicContract {
```